data/_ui/interfaces.txt

```diff
@@ -1376,6 +1376,19 @@ interface "map buttons" bottom right
 		from -50 -90 to -80 -120
 		size 18
 
+	visible if "!is starry"
+	sprite "ui/star"
+		from 0 -120 to -49 -169
+	button ] "_"
+		from 0 -120 to -49 -169
+		size 18
+	visible if "is starry"
+	sprite "ui/ring"
+		from 0 -120 to -49 -169
+	button [ "_"
+		from 0 -120 to -49 -169
+		size 18
+
```

Comment:
It seems like the sprites used for the zoom buttons already match this one, you don't need to add any more sprites at all, just add the star and ring icons as their own images instead of pre-applying them to the button sprite.